public/views/chat.html

```diff
@@ -0,0 +1,57 @@
+<div class="wrapper">
+    <div class="panel">
+        <i class="fa fa-2x fa-comments-o" size="30" aria-hidden="true"></i>
+        <span ng-show="chat.unreadMessageCount === 0" class="text-center">Kissa Chat</span>
+        <span ng-hide="chat.unreadMessageCount === 0" class="text-center">
+            <span class="badge badge-danger">{{chat.unreadMessageCount}}</span>&nbsp;New&nbsp;Message</span>
+        <i class="fa fa-2x fa-chevron-up minimize" ng-click="chat.showHide()" ng-hide="chat.isHidden" aria-hidden="true"></i>
```

Comment:
I see what you did there 😉 

You can avoid the repetition still by doing this.

```html
<i  
  class="fa fa-2x minimize {{chat.isHidden ? 'fa-chevron-up' : 'fa-chevron-down'}}" 
  ng-click="chat.showHide()" 
  aria-hidden="true">
</i>
```